src/linker.rs

```diff
@@ -518,6 +532,11 @@ impl llvm::LLVMDiagnosticHandler for Linker {
     fn handle_diagnostic(&mut self, severity: llvm_sys::LLVMDiagnosticSeverity, message: &str) {
         match severity {
             llvm_sys::LLVMDiagnosticSeverity::LLVMDSError => {
+                // TODO(https://reviews.llvm.org/D155894): Remove this when LLVM no longer emits
+                // these errors.
+                if message.ends_with(": A call to built-in function 'memset' is not supported.\n") {
```

Comment:
What about memcmp, memmove bcmp and strlen?

Reply:
Where's the authoritative list?

Reply:
https://github.com/rust-lang/compiler-builtins/blob/master/src/mem/mod.rs

Reply:
Done.